From 4a7a5305b8c0fe5d9d0434d5831a9f6125692ea9 Mon Sep 17 00:00:00 2001
From: Kuppuswamy Sathyanarayanan <sathyanarayanan.kuppuswamy@intel.com>
Date: Sat, 10 May 2014 11:52:03 -0700
Subject: iio: light: jsa1212: Fix default ALS window

Changed default ALS interrupt window from 0 to 1.
After this change by default, once iio events are
enabled,  ALS will generate an interrupt for every 4
conversion outside the preset window.

Signed-off-by: Kuppuswamy Sathyanarayanan <sathyanarayanan.kuppuswamy@intel.com>
---
 drivers/iio/light/jsa1212.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/iio/light/jsa1212.c b/drivers/iio/light/jsa1212.c
index 9d3c61f..5ee759e 100644
--- a/drivers/iio/light/jsa1212.c
+++ b/drivers/iio/light/jsa1212.c
@@ -1066,7 +1066,7 @@ static int jsa1212_chip_init(struct jsa1212_data *data)
 	data->als_thresh[JSA1212_THRESH_HIGH] = JSA1212_ALS_TH_MAX;
 	data->pxs_thresh[JSA1212_THRESH_LOW] = JSA1212_PXS_TH_MIN;
 	data->pxs_thresh[JSA1212_THRESH_HIGH] = JSA1212_PXS_TH_MAX;
-	data->als_prst_val = JSA1212_INT_ALS_PRST_1CONV;
+	data->als_prst_val = JSA1212_INT_ALS_PRST_4CONV;
 	data->pxs_prst_val = JSA1212_INT_PXS_PRST_1CONV;
 	data->flags = 0x00;
 	data->als_rng_idx = JSA1212_ALS_RNG_0_2048;
-- 
1.7.9.5

